### ISSUE TITLE
release_summary.release_check is empty

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/kingfisher-summarize#176]
For example, this query returns `NULL`:

```sql
select
	distinct release_check
from
	view_data_collection_1563_1564_1565.release_summary;
```

But this query returns 13,502:

```sql
select
	count(distinct cove_output)
from
	release_check
join release on
	release_check.release_id = release.id
where
	collection_id = 1563;
```

A fix isn't urgent as we can query the `release_check` table directly as a workaround.

**Comment by jpmckinney** [comment]
The logic is unchanged in the new code, so it must be an old bug.

Old code:

https://github.com/open-contracting/kingfisher-summarize/blob/483ee9f485a637044f2aea707285e2c16f304316/sql/008-release.sql#L450-L463

New code:

https://github.com/open-contracting/kingfisher-summarize/blob/0b3fa8d44a189561b046ae06fb7e977ad8e3b032/sql/middle/release_summary.sql#L57-L70

**Comment by jpmckinney** [comment]
Closed by #178
